std/file.d

```diff
@@ -4930,7 +4930,8 @@ alias DirIterator = _DirIterator!dip1000Enabled;
         $(LREF DirEntry).
 
     Throws:
-        $(LREF FileException) if the directory does not exist.
+        * $(LREF FileException) if the *path* directory does not exist or read permission is denied.
```

Comment:
The documentation tester seems to be failing for unknown reasons, but I suspect that starting the line with a * is not a good idea.

Reply:
I replaced that, DAutoTest still failing but with a different error now (maybe unrelated). I think DAutoTest did actually pass when I created this pull.